Not having Discord installed breaks game

I experienced this too:

```
15:05:33.485 [Hardware legend.game.Main:81] ERROR: Stack trace:
java.lang.RuntimeException: java.io.FileNotFoundException: \\.\pipe\discord-ipc-0 (The system cannot find the file specified)
        at de.jcm.discordgamesdk.Core.<init>(Core.java:129)
        at discord.DiscordRichPresence.init(DiscordRichPresence.java:37)
        at legend.core.GameEngine.start(GameEngine.java:258)
        at legend.game.Main.main(Main.java:41)
        at legend.game.MainWindows.main(MainWindows.java:38)
Caused by: java.io.FileNotFoundException: \\.\pipe\discord-ipc-0 (The system cannot find the file specified)
        at java.base/java.io.RandomAccessFile.open0(Native Method)
        at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
        at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
        at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
        at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:137)
        at de.jcm.discordgamesdk.impl.channel.WindowsDiscordChannel.<init>(WindowsDiscordChannel.java:24)
        at de.jcm.discordgamesdk.Core.getDiscordChannel(Core.java:55)
        at de.jcm.discordgamesdk.Core.<init>(Core.java:121)
        ... 4 more
15:05:33.487 [Hardware legend.game.Main:83] ERROR: Please copy this crash log and send it to us in the Player Help channel in the Legend of Dragoon Discord server.
15:05:33.487 [Hardware legend.game.Main:84] ERROR: https://discord.gg/legendofdragoon
2025-06-28T19:05:33.489565500Z Hardware INFO Stopping configuration XmlConfiguration[location=C:\Users\Alec-\Downloads\Severed_Chains_Windows2\log4j2.xml, lastModified=2025-06-28T19:04:07.325Z]...
2025-06-28T19:05:33.491074500Z Hardware INFO Configuration XmlConfiguration[location=C:\Users\Alec-\Downloads\Severed_Chains_Windows2\log4j2.xml, lastModified=2025-06-28T19:04:07.325Z] stopped.
```

--- a/src/main/java/discord/DiscordRichPresence.java
+++ b/src/main/java/discord/DiscordRichPresence.java
@@ -32,7 +32,9 @@ public final class DiscordRichPresence {
   public void init() {
     final CreateParams params = new CreateParams();
     params.setClientID(1385814687458918400L); //App ID
-    params.setFlags(CreateParams.getDefaultFlags());
+
+    final long paramFlags = CreateParams.Flags.toLong(CreateParams.Flags.DEFAULT, CreateParams.Flags.SUPPRESS_EXCEPTIONS);
+    params.setFlags(paramFlags);
 
     this.core = new Core(params);
     this.core.setLogHook(LogLevel.ERROR, (level, string) -> LOGGER.error(string));

